.env

```diff
@@ -1,2 +1,5 @@
 ADMIN_NAME=admin
-ADMIN_PASSWORD=password
\ No newline at end of file
+ADMIN_PASSWORD=password
```

Comment:
This shouldn't be in your git repository, this file should be added to your .gitignore and then deleted from git